### PR TITLE
Add overall cloudserver status alert

### PR DESCRIPTION
**Pull request by francoisferrand**
Issue: CLDSRV-75

### CODE REVIEW

File: monitoring/alerts.yaml [hunk 1]
```diff
@@ -6,11 +6,31 @@ x-inputs:
...
+    Expr: sum(up{namespace="${namespace}", service="${service}") < ${replicas}
+    For:   "30s"
+    Labels:
+      severity: warning
+    Annotations:
+      description: "Less than 100% of S3 endpoints are up and healthy"
+      summary: "Data Access service is degraded",
+
+  - alert: DataAccessS3EndpointCritical
+    Expr: sum(up{namespace="${namespace}", service="${service}") * 2 < ${replicas}
```
Comment (lavigne958) [inline monitoring/alerts.yaml]: wrong syntax... 
you miss the closing `}` at the end of the metric labels

**Review by lavigne958 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
A small formatting issue in the `summary` lines: `summary: "Data Access service is degraded",`

The comma is to be deleted I think! Lines 23 and 32.

**Review by ghost — APPROVED** [review]

**Pull request merged**